ThePortugueseMan.HabitTracker/Habit_DataBase/DataBaseViews.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.Data.Sqlite;
+
+namespace DataBaseLibrary;
+
+public class DataBaseViews
+{
+    static string connectionString = @"Data Source=habit-Tracker.db";
+
+    //returns the amount of log in a giver year (YY format)
```

Comment:
Please check the necessity of this comment. 😊

Reply:
Should I add information about the format somewhere else, or is there no need to do so?

Reply:
@ThePortugueseMan I don't think it's needed. It can be seen in the code. 😊